snowpack/src/util.ts

```diff
@@ -70,6 +70,24 @@ export function deleteFromBuildSafe(dir: string, config: SnowpackConfig) {
   return rimraf.sync(dir);
 }
 
+const nodeModulesPackageName = new RegExp(`${path.sep}node_modules${path.sep}(.+?)(?=${path.sep})${path.sep}`);
+export function ignoreNodeModulesFromMount(mount: Record<string, MountEntry> = {}) {
+  if (!mount) {
+    return [];
+  }
+
+  try {
+    return Object.keys(mount)
```

Comment:
Depending how many packages are being included in the mount, this might end up with duplicates, ex: `['**/node_module/astro/**', '**/node_module/astro/**']`.

If for some reason this is a concern we can convert the array to a set and then back to an array so we end up with unique values, ex: `[...new Set(['**/node_module/astro/**', '**/node_module/astro/**'])]`